File: livingrimoire start here/LivinGrimoire java/src/Auxiliary_Modules/PercentDripper.java
package Auxiliary_Modules;

public class PercentDripper {
    private final DrawRnd dr = new DrawRnd();
    private int limis = 35;

    public void setLimis(int limis) {
        this.limis = limis;
    }
    public boolean drip(){
        return dr.getSimpleRNDNum(100) < limis;
    }
    public boolean dripPlus(int plus){
        return dr.getSimpleRNDNum(100) < limis + plus;
    }
}

File: livingrimoire start here/LivinGrimoire Kotlin/kotlin/auxiliary_modules/PercentDripper.kt
package auxiliary_modules

class PercentDripper {
    private val dr = DrawRnd()
    private var limis = 35
    fun setLimis(limis: Int) {
        this.limis = limis
    }

    fun drip(): Boolean {
        return dr.getSimpleRNDNum(100) < limis
    }

    fun dripPlus(plus: Int): Boolean {
        return dr.getSimpleRNDNum(100) < limis + plus
    }
}